.github/workflows/ci.yml

```diff
@@ -1,17 +1,81 @@
 name: CI
 
-on: [push]
+on: push
 
 jobs:
-  build:
+  webknossos_linux:
     runs-on: ubuntu-latest
     strategy:
       max-parallel: 4
       matrix:
         python-version: [3.7, 3.8]
+    defaults:
+      run:
+        working-directory: webknossos
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+    - uses: marceloprado/has-changed-path@v1
+      id: changed
+      with:
+        paths: webknossos
```

Comment:
cool stuff :+1: however, this is still using `marceloprado` (you mentioned in https://github.com/scalableminds/webknossos-libs/pull/398#discussion_r699275399 that you want to change this?).

Reply:
This happened already in the meantime, see my last changes